packages/plugins/deploy-tracker/src/trackingFunctions.js

```diff
@@ -111,6 +111,44 @@ export default class TrackingFunctions {
     cb();
   }
 
+  async trackMigrationScript(params, callback) {
+    if (!this.enabled) {
+      return callback();
```

Comment:
I think you should always callback with the `params` as second argument, because if there is another action registered, it will not get the `params`

Reply:
Good catch! I'll have to refactor this away again anyways, due to the other comment that states scripts-runner should take care of tracking itself.